shell.nix

```
@@ -0,0 +1,16 @@
+let
+  moz_overlay = import (builtins.fetchTarball https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz);
+  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
+in
+  with nixpkgs;
+  stdenv.mkDerivation {
+    name = "ord-shell";
+    buildInputs = [
+      # to use the latest nightly:
+      nixpkgs.latest.rustChannels.stable.rust
+      clang
+      just
+      # to use the project's rust-toolchain file:
+      # (nixpkgs.rustChannelOf { rustToolchain = ./rust-toolchain; }).rust
+    ];
+  }
```

Is clang needed? I assume it is, probably due to a dependency somewhere, but wanted to make sure.

How about removing the comment about latest nightly, and also about the rust-toolchain file, since we don't have one?

```suggestion
let
    moz_overlay = import (builtins.fetchTarball https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz);
  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
in
  with nixpkgs;
  stdenv.mkDerivation {
    name = "ord-shell";
    buildInputs = [
      nixpkgs.latest.rustChannels.stable.rust
      clang
      just
    ];
  }
```

---

thanks, that was sloppy. i'm a little unclear on what rust depends on, but yea it will get pulled in via the rust dep.